Foundation/NSDictionary.swift

```diff
@@ -264,7 +264,17 @@ public class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCodin
         return matching
     }
 
-    public override var description: String { NSUnimplemented() }
+    public override var description: String {
+        get {
+            var _description: String = "{\n"
+            for key in self.allKeys {
+                _description += "    \(key) = \(objectForKey(key)!);\n"
```

Comment:
@phausler also what would the expected behaviour be in this situation if someone has done something like make a dictionary of dictionaries with *many* levels of depth?  Some kind of termination at some point?